### ISSUE TITLE
Pull and Push 

### ISSUE THREAD

**Issue by aarnaud** [alberthier/git-webui#7]
Hi,

Can'i use push and pull (to github for exemble) with web interface ?

When there is a conflict, have you got a diff view for resolve this conflict?

Good job :-)

Thank you

**Comment by alberthier** [comment]
Yes, agreed, a conflict resolution UI would definitively be helpful in everyday's use.

**Comment by iurly** [comment]
Hi, what's the status of this feature?
It looks like there's not even a "fetch" button, am I right?
Thanks!

**Comment by bruno-nk** [comment]
Is this project still alive? It looks really cool and seems to work nicely but without being able to pull from a remote repository it won't be usable for me :(
